FZ6D-13_product_fix: Neon effect + Collapsing Text

andrzejpuzynski opened the pull request:
Neon Effect 
Collapsing Text
Fixing margins

andrzejpuzynski committed:
Fixed product.

product.html
- adding collapse;

product.css
- adding cursor pointer;
- adding hover neon effect;

andrzejpuzynski committed:
Fixed product 2nd.

product.html
- adding div neon;

product.css
- dead code remove;
- dead class remove;
- adding cursor pointer;
- adding hover;
- adding neon effect;

andrzejpuzynski committed:
Fixed product 3nd.

product.css
- fixing margins in: product_item;
- fixing margins in: product_text;
- changing h4;

andrzejpuzynski committed:
Fixed product 4nd.

product.html
- adding non breaking spaces to text;

product.css
- changing margins in product_text class;

Zetriax reviewed (dismissed):
Good job +1

tomson7777 commented on product.html:
```diff
@@ -22,7 +22,7 @@
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>ŃŃ
```
to ŃŃ ma być w linijce 25? czy chochlik się wdarł:)

andrzejpuzynski replied:
Teraz już nie jest potrzebne :)

tomson7777 commented on product.html:
```diff
@@ -44,50 +44,47 @@
-                        <p id="speedAndClarity" class="lead text-left product-text collapse">Wyświetlone wydarzenia zawierają krótki opis nazwy, czasu i&nbsp;miejsca, dzięki czemu łatwo i szybko znajdziesz&nbsp;to, co&nbsp;Cię interesuje.
+                        <div class="neon"><i class="fa fa-eye fa-5x"></i>
+                        <h4 class="text-center">Szybkość i czytelność</h4></div>
+                        <p id="speedAndClarity" class="lead text-left product-text collapse">Wyświetlone wydarzenia zawierają krótki opis nazwy, czasu i&nbsp;miejsca, dzięki czemu łatwo i&nbsp;szybko znajdziesz&nbsp;to, co&nbsp;Cię interesuje.
```
speedAndClarity - > speed-and-clarity, używamy kebab-case w html, camelCase do JS

robert-jemiolo replied:
w kilku miejscach by się przydało zmienić

tomson7777 commented on product.html:
```diff
@@ -44,50 +44,47 @@
-                        <div class="product-title" ><h4 class="text-center">Szybkość i czytelność</h4></div>
-                        <p id="speedAndClarity" class="lead text-left product-text collapse">Wyświetlone wydarzenia zawierają krótki opis nazwy, czasu i&nbsp;miejsca, dzięki czemu łatwo i szybko znajdziesz&nbsp;to, co&nbsp;Cię interesuje.
+                        <div class="neon"><i class="fa fa-eye fa-5x"></i>
+                        <h4 class="text-center">Szybkość i czytelność</h4></div>
```
poprawiłbym ułożenie tagów, tak aby div był domknięty pod sobą, tzn:
```
<div>
  <i></i>
  <h4></h4>
</div>
```

andrzejpuzynski replied:
Jasne, będzie czytelniej.

tomson7777 commented on product.html:
```diff
@@ -44,50 +44,47 @@
+                                <ul class="text-left product-text list-unstyled">
+                                    <li class="product-list"><i class="fa fa-check-square-o fa-lg"></i>&nbsp;<span class="text-uppercase">Data od do</span> - pozwala wyświetlić wydarzenia odbywające&nbsp;się w&nbsp;określonym przedziale czasu.</li>
+                                    <li class="product-list"><i class="fa fa-check-square-o fa-lg"></i>&nbsp;<span class="text-uppercase">Typ biletu</span> - pozwala wyświetlić tylko wydarzenia z&nbsp;wymaganym biletem wstępu lub z&nbsp;wstępem wolnym.</li>
+                                    <li class="product-list"><i class="fa fa-check-square-o fa-lg"></i>&nbsp;<span class="text-uppercase">Kategoria</span> - umożliwia wyświeltenie wydarzeń należących do&nbsp;jednej z&nbsp;zadanych kategorii, np.&nbsp;teatr, kino, koncert itp.</li>
```
unikamy tak długich linii, tutaj jest z dobre 270 znaków. Niech będzie chociaż
```
<li>
  <i></i> 
  <span></span> TEXT
</li>
```

andrzejpuzynski replied:
Tak, zdecydowanie jest czytelniej :)

tomson7777 commented on product.html:
```diff
@@ -44,50 +44,47 @@
                     <div class="col-md-6 col-sm-12 col-xs-12 prod_item" data-toggle="collapse" data-target="#filters">
-                            <div><i class="fa fa-check-square-o fa-5x"></i></div>
-                            <div class="product-title"><h4 class="text-center">Filtry przeglądania</h4></div>
+                            <div class="neon"><i class="fa fa-check-square-o fa-5x"></i>
```
o jeden tabulator za dużo, od linii 60.

andrzejpuzynski replied:
Divy poprawione

tomson7777 reviewed (commented):
ładnie! drobne uwagi co do formatowania

robert-jemiolo commented on product.html:
```diff
@@ -44,50 +44,47 @@
-            Opis może być rozwijany-->
             <div class="product-container">
                 <hr>
                 <h4>Planowane funkcje produktu</h4>
```
może zamiast hrów dać border top i bottom w css?

andrzejpuzynski replied:
Na kolejnym etapie dynamizacji strony wezmę to pod uwagę.

andrzejpuzynski committed:
Adding Code Review fixes:

- fixing tab's positions,
- adding some div conteners,
- changing camelCase to kebab-case in #id-names.

tomson7777 reviewed (commented):
approve

tomson7777 reviewed (approved):
approve
